front/app/modules/commercial/analytics/admin/utils/timeSeries.ts

```diff
@@ -0,0 +1,230 @@
+import moment, { Moment } from 'moment';
+import { IResolution } from 'components/admin/ResolutionControl';
+
```

Comment:
Made the logic that parses time series more reusable using this higher-order function